agimus_demo_01_lfc_alone/dependencies.repos

```diff
@@ -1,30 +1,15 @@
 repositories:
-  vcs_repos/linear_feedback_controller_msgs:
-    type: git
-    url: https://github.com/loco-3d/linear-feedback-controller-msgs
-    version: 0.1.3
-
-  vcs_repos/coal:
-    type: git
-    url: https://github.com/coal-library/coal
-    version: v3.0.0
-
-  vcs_repos/pinocchio:
-    type: git
-    url: https://github.com/stack-of-tasks/pinocchio
-    version: v3.3.0
-
-  vcs_repos/colmpc:
+  vcs_repos/franka_description:
     type: git
-    url: https://github.com/agimus-project/colmpc
-    version: v0.2.0  # TODO: change to new version tag
+    url: https://github.com/agimus-project/franka_description
+    version: humble_devel
 
-  vcs_repos/mim_solvers:
+  vcs_repos/linear_feedback_controller:
     type: git
-    url: https://github.com/machines-in-motion/mim_solvers
-    version: v0.0.3
+    url: https://github.com/loco-3d/linear-feedback-controller
+    version: humble-devel
```

Comment:
we need the url adress for the secret branch

Reply:
This will be merged very soon, for now go to vcs_repos/linear-feedback-controller and checkout [this branch](https://github.com/MaximilienNaveau/linear-feedback-controller/tree/topic/mnaveau/humble-devel/gravity_compensation_and_vel_filter)